src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,19 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable } from '@nestjs/common';
+import { jwtConstants } from './constants';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+  constructor() {
+    super({
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      ignoreExpiration: false,
+      secretOrKey: jwtConstants.secret
+    });
+  }
+
+  async validate(payload: any) {
+    return { id: payload.id };
+  }
```

Comment:
```suggestion
  async validate(payload: any) {
    return this.authService.getLoggedUser(payload.id);
  }
```

 Phải validate xem user có tồn tại ko. Đồng thời user được return sẽ được jwt gắn vào request.user